discovery-provider/chain/health.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+health_url = "http://localhost:8545/health"
+rpc_url = "http://localhost:8545"
+
+health_res = requests.get(health_url)
+health_json = health_res.json()
+
+healthy = True
+
+health_check_node = health_json.entries["node-health"]
+health_check_description = health_check_node.description
+health_check_healthy = health_json.status == "Unhealthy"
+
+if health_check_healthy:
+    healthy = False
+
+not_producing_msg = "The node stopped producing blocks"
+if health_check_healthy and not_producing_msg in health_check_description:
```

Comment:
what if a signer stops producing blocks? i think "The node stopped producing blocks" appears in the health check description then too